### PR TITLE
Added count for Cpp

### PR DESCRIPTION
**Pull request by RashmitTopG**
### Description

Added term entry for cpp.count()

### Issue Solved

#5475

Added a new term entry for cpp.count() in unordered_map

### Type of Change

- Adding a new entry

### Checklist

- [✓] All writings are my own.
- [✓] My entry follows the Codecademy Docs style guide.
- [✓] My changes generate no new warnings.
- [✓] I have performed a self-review of my own writing and code.
- [✓] I have checked my entry and corrected any misspellings.
- [✓] I have made corresponding changes to the documentation if needed.
- [✓] I have confirmed my changes are not being pushed from my forked `main` branch.
- [✓] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [✓] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by RashmitTopG** [comment]
@cigar-galaxy82 can you review this PR ??

File: content/cpp/concepts/unordered-map/terms/count/count.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        } else {
+            std::cout << mammal << " does not exist in the map.\n";
+        }
+    }
+
+    return 0;
+}
+```
+
+## Codebyte Example 2
```
Comment (cigar-galaxy82) [inline content/cpp/concepts/unordered-map/terms/count/count.md:61]:
```suggestion
## Codebyte Example
```

File: content/cpp/concepts/unordered-map/terms/count/count.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+The following codebyte example demonstrates the use of the .count() method with an unordered_map to check for the presence of various fruits and their prices:
```
Comment (cigar-galaxy82) [inline content/cpp/concepts/unordered-map/terms/count/count.md:63]:
```suggestion
The following codebyte example demonstrates the use of the `.count()` method with an unordered map to check for the presence of various fruits and their prices:
```

**Review by cigar-galaxy82 — CHANGES_REQUESTED** [review]

**Comment by cigar-galaxy82** [comment]
Hi @RashmitTopG thank you for contributing to Codecademy docs, I have suggested a few changes please modify them.

**Review by mamtawardhani — APPROVED** [review]
Thank you for making the required changes and contributing to Codecademy Docs @RashmitTopG 😄 

The entry looks good for a second round of review! 🚀

File: content/cpp/concepts/unordered-map/terms/count/count.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
+---
+Title: '.count()'
+Description: 'Returns the number of elements with the specified key.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Objects'
+  - 'OOP'
+  - 'Classes'
```
Comment (Radhika-okhade) [inline content/cpp/concepts/unordered-map/terms/count/count.md:10]:
```suggestion
  - 'Classes'
  - 'Objects'
  - 'OOP'
```

File: content/cpp/concepts/unordered-map/terms/count/count.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+        } else {
+            std::cout << mammal << " does not exist in the map.\n";
+        }
+    }
+
+    return 0;
+}
+```
+
+The code above produces the output as follows:
```
Comment (Radhika-okhade) [inline content/cpp/concepts/unordered-map/terms/count/count.md:59]:
```suggestion
This example results in the following output:
```

**Review by Radhika-okhade — APPROVED** [review]

**Pull request merged**